config.yaml

```
@@ -4,19 +4,67 @@ title: Code for Germany
 theme: codefor-theme
 defaultContentLanguage: de
 
-params:
-  twitter: "https://twitter.com/codeforde"
-  twitter_handle: "@codeforde"
-  github: "https://github.com/okfde"
-  author: "codeforde"
-  keywords:
-    - opendata
-    - codeforde
-    - Open Government
-    - Civic Tech
-    - Free and Open Source Software
-    - FOSS
-    - Digitales Ehrenamt
+params:
+  twitter: "https://twitter.com/codeforde"
+  twitter_handle: "@codeforde"
+  github: "https://github.com/okfde"
+  author: "codeforde"
+  keywords:
+    - opendata
+    - codeforde
+    - Open Government
+    - Civic Tech
+    - Free and Open Source Software
+    - FOSS
+    - Digitales Ehrenamt
+
+languages:
+  de:
+    languageName: Deutsch
+    weight: 1
+  en:
+    languageName: English
+    weight: 2
+
+menu:
+  main:
+  - identifier: about
+    name: Über uns
+    url: /about/
+    weight: 1
+  - identifier: mitmachen
+    name: Mitmachen
+    url: /mitmachen
+    weight: 2
+  - identifier: ziele
+    name: Ziele
+    url: /ziele
+    weight: 3
+  - identifier: projekte
+    name: Projekte
+    url: /Projekte/
```

```suggestion
    url: /projekte/
```
vielleicht? Oder hat das einen Grund, warum es hier groß ist?